packages/lesswrong/lib/sql/UpdateQuery.ts

```diff
@@ -186,6 +186,9 @@ class UpdateQuery<T extends DbObject> extends Query<T> {
       const resolvedField = this.resolveFieldName(field);
       return format(resolvedField, updateValue);
     } catch (e) {
+      if (this.table instanceof Table && this.table.hasResolverOnlyField(field)) {
```

Comment:
This is needed to allow us to include the resolver-only field in the post edit form, but to exclude if from the postgres query to update the object, which would be a syntax error.